Gopkg.lock

```diff
@@ -176,6 +182,12 @@
   version = "v1.0.0"
 
 [[projects]]
+  name = "github.com/stretchr/testify"
```

Comment:
Does it really make sense to pull in `testify` when it's only used here once?! Yes, because we want to rewrite all the test!